scripts/vscripts/statcollection/lib/statcollection.lua

```diff
@@ -411,9 +424,10 @@ function statCollection:sendStage3(winners, lastRound)
 end
 function statCollection:submitRound(args)
     --We receive the winners from the custom schema, lets tell phase 3 about it!
-    returnArgs = self.custom:submitRound(args)
+    returnArgs = customSchema:submitRound(args)
```

Comment:
Wouldn't this send every round each time it is called?

I don't see where the round data is nuked, and the roundID kept to increment from

i.e. 

* first invocation... round0 data
* second invocation... round0 + round1 data
* third invocation... round0 + round1 + round2 data